org.small_tech.Gnomit.appdata.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
```

Comment:
not a blocker, but the appdata/desktop files should go upstream

Reply:
Just to confirm, so should I simply remove the following files from the repo:

  - org.small_tech.Gnomit.appdata.xml
  - org.small_tech.Gnomit.desktop

Or is there something special I need to do to ensure these files “go upstream”. By upstream, I’m assuming you mean the app itself?

Happy to remove the files if they’re not necessary. 

Thanks!

Reply:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
@aral, what @bilelmoussaoui meant was that those files, ideally, should be part of your release; thus it is Gnomit should bundle them.